core/Tracker/Db/Mysqli.php

```diff
@@ -52,6 +60,17 @@ public function __construct($dbInfo, $driverName = 'mysql')
         $this->username = $dbInfo['username'];
         $this->password = $dbInfo['password'];
         $this->charset = isset($dbInfo['charset']) ? $dbInfo['charset'] : null;
+
+
+        $this->enable_ssl = $dbInfo['enable_ssl'];
+        $this->ssl_key = $dbInfo['ssl_key'];
+        $this->ssl_cert = $dbInfo['ssl_cert'];
+        $this->ssl_ca = $dbInfo['ssl_ca'];
+        $this->ssl_ca_path = $dbInfo['ssl_ca_path'];
+        $this->ssl_cipher = $dbInfo['ssl_cipher'];
+        $this->ssl_no_verify = $dbInfo['ssl_no_verify'];
```

Comment:
`$dbInfos` may not have all of these entries since some are optional, so we'll have to surround each line in a `!empty()` if like:

```
if (!empty($dbInfo['enable_ssl'])) {
    $this->enable_ssl = $dbInfo['enable_ssl'];
}
```

Reply:
👍